apps/backend/src/libs/modules/controller/libs/types/controller-route-parameters.type.ts

```diff
@@ -2,11 +2,13 @@ import { type HTTPMethod } from "~/libs/modules/http/http.js";
 import { type ValidationSchema } from "~/libs/types/types.js";
 
 import { type APIHandler } from "./api-handler.type.js";
+import { type APIPreHandler } from "./api-pre-handler.type.js";
 
 type ControllerRouteParameters = {
 	handler: APIHandler;
 	method: HTTPMethod;
 	path: string;
+	preHandler?: APIPreHandler[];
```

Comment:
we our property we always know it will be an array, so it should be plural
```suggestion
	preHandlers?: APIPreHandler[];
```